src/@types/clusters.d.ts

```diff
@@ -0,0 +1,35 @@
+interface IClusterCapabilities {
+  hasOptionalIngress: boolean;
+  supportsHAMasters: boolean;
+}
+
+interface IBaseCluster {
+  id: string;
+  api_endpoint: string;
+  create_date: string | null;
+  owner: string;
+  name: string;
+  release_version: string;
+
+  credential_id?: string;
+  delete_date?: string | null;
+
+  // Injected by client-side.
+  capabilities?: IClusterCapabilities;
+  keyPairs?: import('shared/types').IKeyPair[];
```

Comment:
A bit weird, but if we would use regular ES module imports, this whole file is treated like an ES module